portal_blocks_documentation/docs.py

```diff
@@ -42,7 +45,12 @@ def ensure_index_md(block_name: str):
             "draft: false",
             f"geekdocFilePath: portal_blocks/{block_name}/_index.md",
             "---",
-            f"# {block_name}"
+            f"# {block_name}",
+            '{{< include file="' + MOUNT_DIR + '/' + block_name + '/docs/official.md" >}}',
+            "",
+            f"# Community Additions",
```

Comment:
Community additions feels kinda weird :thinking:

Reply:
I see... you used "Extra Documentation" in the layout